app/controllers/api/v2/courses_controller.rb

```diff
@@ -52,6 +73,27 @@ def build_course
         @course = @provider.courses.find_by!(course_code: params[:code].upcase)
         authorize @course
       end
+
+      def update_params
+        params
+          .require(:course)
+          .except(:id, :type)
+          .permit(
+            :about_course,
+            :course_length,
+            :fee_details,
+            :fee_international,
+            :fee_uk_eu,
+            :financial_support,
+            :how_school_placements_work,
+            :internal_server_error,
```

Comment:
internal_server_error
?

permit as a a param?

Reply:
Ya, wat? I don't know how that got in there, will double-check and remove. Thanks!

Reply:
done